marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/writer/HubDataWriterFactory.java

```diff
@@ -53,7 +53,7 @@ public HubDataWriterFactory(Map<String, String> params, StructType schema) {
         if (logger.isDebugEnabled()) {
             logger.debug("Creating DataWriter with taskId: " + taskId);
         }
-        return new HubDataWriter(hubClient, taskId, schema, params);
+        return new HubDataWriter(hubClient, schema, params);
```

Comment:
I think that since taskId doesn't matter, let's remove the debug logging above it. It would be meaningless to see it.